### ISSUE TITLE
db_field fails on EmbeddedDocumentField

### ISSUE THREAD

**Issue by wpjunior** [hmarr/mongoengine#523]
the test case:

``` python
from mongoengine import *
from mongoengine import register_connection

register_connection('default', name='test')


class EDocument(EmbeddedDocument):
    val = IntField()

class TestDocument(Document):
    e = EmbeddedDocumentField(EDocument,
                              db_field='Eb')

TestDocument.drop_collection()

doc = TestDocument(e=EDocument(val=15))
doc.save()
del doc

doc = TestDocument.objects.first()
print doc.validate()
```

**Comment by wpjunior** [comment]
raises a:

``` python
mongoengine.base.ValidationError: Errors encountered validating document:
e: Invalid embedded document instance provided to an EmbeddedDocumentField ("e")
```

**Comment by rozza** [comment]
Fixed in 0.6.13
